include/pressio/utils/logger/spdlog/sinks/systemd_sink.hpp

```diff
@@ -79,7 +79,7 @@ class systemd_sink : public base_sink<Mutex>
 
     int syslog_level(level::level_enum l)
     {
-        return syslog_levels_.at(static_cast<levels_array::size_type>(l));
+        return syslog_levels_.at(static_cast<levels_array::ordinal_type>(l));
```

Comment:
why this? this should have nothing to do with traits

Reply:
Thanks for catching this one! Fixed on e08a2ff261e6b814dac417be7799555ef336d618.

Should this file be in [`unused`](https://github.com/Pressio/pressio/tree/356-new-container-traits/include/pressio/utils/logger/spdlog/sinks/unused) folder ? It doesn't seem to be included anywhere.

Reply:
at some point we need to revise all the logging but that should be done independently